docusaurus/docs/how-to-guides/ui-extensions/register-an-extension.md

```diff
@@ -37,6 +39,8 @@ export const plugin = new AppPlugin().addLink({
 
 ### Hide a link in certain conditions
 
+You can hide the link at the extension point. For example:
```

Comment:
change "the link" to "a link" as it shows only removing a certain link extension (time series

Reply:
done